kmm/kmm-post-data/src/commonTest/kotlin/com/gchristov/newsfeed/kmmpost/ReadingTimeCalculatorTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.gchristov.newsfeed.kmmpost
+
+import com.gchristov.newsfeed.kmmpostdata.util.ReadingTimeCalculator
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class ReadingTimeCalculatorTest {
+
+    @Test
+    fun returnsZeroMinutesWhenNoWords() {
+        val totalWordCount = 0;
+        runTest(totalWordCount = totalWordCount, expectedReadingTimeMinutes = 0)
+    }
+
+    @Test
+    fun returnsZeroMinutesWhenNegativeWords() {
+        val totalWordCount = -20;
+        runTest(totalWordCount = totalWordCount, expectedReadingTimeMinutes = 0)
+    }
+
+    @Test
+    fun takesMinimumOneMinuteWhenLessThanTwoHundredWords() {
+        // given
+        // 1 minute is 200 words
+        val totalWordCount = 50;
+        runTest(totalWordCount = totalWordCount, expectedReadingTimeMinutes = 1)
+    }
+
+    @Test
+    fun takesMultipleMinutesForExactlyTwoHundredWords() {
```

Comment:
Nitpick: Maybe this could be `takesOneMinuteForExactlyTwoHundredWords` since it's what we are testing? We could then have another test that checks 3 minutes for example by using `600` words and that one could be `takesMultipleMinutesForExactlyTwoHundredWords`